tests/e2e/default/translations/enabled-languages.wdio-spec.js

```diff
@@ -0,0 +1,110 @@
+const isEqual = require('lodash/isEqual');
+const utils = require('../../../utils');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+
+describe('Enabling/disabling languages', () => {
+  before(async () => {
+    const settings = {
+      languages: [
+        {
+          locale: 'en',
+          enabled: true,
+        },
+        {
+          locale: 'es',
+          enabled: true,
+        },
+        {
+          locale: 'fr',
+          enabled: true,
+        },
+      ],
+    };
+    await utils.updateSettings(settings, true);
+    await browser.waitUntil(async () => {
```

Comment:
I was seeing some runs failing because the login page didn't have the expected languages where this bit of code helped stabilize the test but I can't reproduce it anymore. I'm getting rid of it!